### PR TITLE
Add buttons for FAQ and Plugins

### PR DESCRIPTION
**Pull request by X9VoiD**
This makes it easier for people to reach places that are *not* a single click away

### CODE REVIEW

File: OpenTabletDriver.Native/SystemInfo.cs [hunk 1]
```diff
@@ -42,5 +42,22 @@ public static void Open(string path)
...
+                    Process.Start("explorer.exe", path);
+                    break;
+                case RuntimePlatform.Linux:
+                    Process.Start("dbus-send", $"--session --dest=org.freedesktop.FileManager1 --type=method_call /org/freedesktop/FileManager1 org.freedesktop.FileManager1.ShowFolders array:string:\"file:{path}\" string:\"\"");
+                    break;
+                case RuntimePlatform.MacOS:
+                case RuntimePlatform.FreeBSD:
+                    Process.Start("open", $"{path}");
+                    break;
+            }
```
Comment (InfinityGhost) [inline OpenTabletDriver.Native/SystemInfo.cs:60]: This shouldn't be required, `Open(string path)` should handle MIME types such as directories and files

**Review by InfinityGhost — CHANGES_REQUESTED** [review]

File: OpenTabletDriver.Native/SystemInfo.cs (hunk 1 above)
Comment (X9VoiD) [reply OpenTabletDriver.Native/SystemInfo.cs:60]: Yeah, opening folders with `Open()` also works on my Windows, but on my Linux it opens up as a VSCode workspace, so I directly used dbus to open a folder using the default file manager

Comment (InfinityGhost) [reply OpenTabletDriver.Native/SystemInfo.cs]: Your MIME types are misconfigured then, it should use your file manager. `xdg-open` for a directory path opens in `nemo` for me like it is supposed to.

Comment (InfinityGhost) [reply OpenTabletDriver.Native/SystemInfo.cs]: Relying on dbus to do this causes problems with i3 and other minimal window managers as well.

Comment (X9VoiD) [reply OpenTabletDriver.Native/SystemInfo.cs]: Change pushed

**Comment by InfinityGhost** [comment]
lg2m

**Review by InfinityGhost — APPROVED** [review]

**Pull request merged**